libmscore/element.h

```diff
@@ -234,7 +235,7 @@ class Element : public QObject, public ScoreElement {
             SYSTEM_DIVIDER,
             STEM_SLASH,
             LINE,
-
+            ANNOTATION,
             ARPEGGIO,
             ACCIDENTAL,
```

Comment:
Oops, it is there already: 
Note: keep in sync with array elementNames[] in element.cpp